skyportal/models.py

```diff
@@ -359,6 +359,52 @@ class Instrument(Base):
                         default=[])
 
 
+class Taxonomy(Base):
+    __tablename__ = 'taxonomy'
+    name = sa.Column(sa.String, nullable=False,
+                     doc='Short string to make this taxonomy memorable '
+                         'to end users.'
+                     )
+    hierarchy = sa.Column(JSONB, nullable=False,
+                          doc='Nested JSON describing the taxonomy '
+                              'which should be validated against '
+                              'a schema before entry'
+                          )
+    provenance = sa.Column(sa.String, nullable=True,
+                           doc='Identifier (e.g., URL or git hash) that '
+                               'uniquely ties this taxonomy back '
+                               'to an origin or place of record'
+                           )
+    version = sa.Column(sa.String, nullable=False,
+                        doc='Semantic version of this taxonomy'
+                        )
+    isLatest = sa.Column(sa.Boolean, default=True, nullable=False,
```

Comment:
What happens if multiple rows in the `taxonomies` table have this flagged?

Reply:
It shouldn't happen since all older taxonomies with that name have this flagged changed upon ingestion. Still, the logic of what to show in case there's a conflict will be handled by the classification apis.